larpandoracontent/LArVertex/TrainedVertexSelectionAlgorithm.cc

```diff
@@ -295,28 +296,49 @@ void TrainedVertexSelectionAlgorithm::GetEventShapeFeatures(const ClusterList &c
         this->UpdateSpanCoordinate(minPosition.GetZ(), maxPosition.GetZ(), zMin, zMax);
     }
 
-    const float xSpan(this->GetCoordinateSpan(xMax, xMin));
-    const float ySpan(this->GetCoordinateSpan(yMax, zMin));
-    const float zSpan(this->GetCoordinateSpan(yMax, zMin));
-
-    // Calculate the volume and longitudinality of the event (ySpan often 0 - to be investigated).
-    if ((xSpan > std::numeric_limits<float>::epsilon()) && (ySpan > std::numeric_limits<float>::epsilon()))
-    {
-        eventVolume     = xSpan * ySpan * zSpan;
-        longitudinality = zSpan / (xSpan + ySpan);
-    }
-
-    else if (ySpan > std::numeric_limits<float>::epsilon())
+    // Calculate the area and longitudinality of the event. Option retained for deprecated implementation.
+    if (m_use2DEventShapes)
     {
-        eventVolume     = ySpan * ySpan * zSpan;
-        longitudinality = zSpan / (ySpan + ySpan);
+      const float xSpan(this->GetCoordinateSpan(xMax, xMin));
+      const float zSpan(this->GetCoordinateSpan(zMax, zMin));
+      
+      if ((xSpan > std::numeric_limits<float>::epsilon()))
+      {
+	eventArea       = xSpan * zSpan;
```

Comment:
Some whitespace and indentation issues, but suggest we merge after the clang format changes have been applied, and then we can simply apply clang format to these updated files too, so these are non-issues now.